### DISCUSSION TITLE
Mi practica de pasteleria

### DISCUSSION THREAD

**Discussion by Jramo5** [Launch-X-Latam/MisionFrontEnd#291] [General]
Hola buenas aqui les comparto mi repositorio de mi practica de la pasteleria.

[REPOSITORIO](https://github.com/Jramo5/Launch-X-Latam-MisionFrontEnd/tree/main/HTML/Pasteleria-Practica)

[PAGINA EN VIVO](https://jramo5.github.io/Launch-X-Latam-MisionFrontEnd/HTML/Pasteleria-Practica/index.html)

Practicas anteriores y futuras ---> [https://jramo5.github.io/Launch-X-Latam-MisionFrontEnd/](https://jramo5.github.io/Launch-X-Latam-MisionFrontEnd/)

Todos los enlaces y subpaginas funcionan

**Comment by zaazilcb** [comment]
¡¡¡Esta excelente!!! una duda, ¿cómo le hiciste para que se deslizaran las imágenes en el inicio?

**Reply by Jramo5** [reply]
Hola, se hizo con la propiedad  [animation](https://developer.mozilla.org/es/docs/Web/CSS/CSS_Animations/Using_CSS_animations) y [@keyframes](https://developer.mozilla.org/es/docs/Web/CSS/@keyframes#:~:text=Resumen,determinados%20puntos%20durante%20la%20animaci%C3%B3n.)

**Reply by zaazilcb** [reply]
gracias!
